apollo-ios/Sources/Apollo/IncrementalJSONResponseParsingInterceptor.swift

```diff
@@ -101,14 +101,15 @@ public struct IncrementalJSONResponseParsingInterceptor: ApolloInterceptor {
           }
         }
 
+        createdResponse._legacyResponse = nil
```

Comment:
I think this is the correct action to take here. There is nothing to set it to other than `nil` because this code path is for incremental responses. I think leaving the initial (partial) response in `_legacyResponse` is also incorrect because it's not accurate. The 'legacy' part of the response logic could also be argued to mean pre-defer-support in which case `nil` again seems the correct choice.